EdFi.Buzz.UI.Angular/yarn.lock

```diff
@@ -6550,7 +6550,7 @@ promise@^7.1.1:
   dependencies:
     asap "~2.0.3"
 
-prop-types@^15.6.2:
+prop-types@^15.6.2, prop-types@^15.7.2:
```

Comment:
Do you know why we have multiple versions of the prop-types? This seems strange. Just wondering if we can have one version here.